### ISSUE TITLE
All Travis CI osx builds will fall due to known upstream issue. 

### ISSUE THREAD

**Issue by bokov** [leeper/rio#242]
The `data.table` team has closed the following ticket with a workaround for an issue someone else reported that seems to be the same as the one we are having:

https://github.com/Rdatatable/data.table/issues/3938#issue-502824551

Please specify whether your issue is about:

 - [x] a possible bug
 - [ ] a question about package functionality
 - [x] a suggested code or documentation change, improvement to the code, or feature request

If you are reporting (1) a bug or (2) a question about code, please supply:

 - [a fully reproducible example](http://stackoverflow.com/questions/5963269/how-to-make-a-great-r-reproducible-example) using a publicly available dataset (or provide your data)
 - if an error is occurring, include the output of `traceback()` run immediately after the error occurs
 - the output of `sessionInfo()`

Put your code here:

```R
## load package
library("rio")

## code goes here


## session info for your system
sessionInfo()
```

**Comment by bokov** [comment]
Made bugfix branch implementing their change to `.travis.yml` and pushed it to my fork. In a few hours we'll see if it fixes the osx data.table error. If it does, I will submit a PR.

**Comment by bokov** [comment]
Travis CI again works for MacOSX!! The developer versions of R still failing for Appveyor. I think maybe Appveyor is not finding the binary versions of 22 packages that `rio` depends on or suggests.

No idea so far what to do about Appveyor, but here's a PR for Travis-CI.

**Comment by bokov** [comment]
Here is the build that's working on Mac again:

https://travis-ci.org/bokov/rio/builds/597576597

**Comment by bokov** [comment]
Closed by #243 , thanks!

**Comment by bokov** [comment]
I am reopening this issue because apparently the pre-built Mac binary for `data.table` is now available, and builds will go faster without having to install LLVM and compile `data.table`. 

I undid my changes from #243 in my fork, and _that_ problem no longer happens in the builds (a different one does, unrelated and possibly at the Travis CI end, I will submit a separate ticket for that one).

Do you prefer to roll back PR #243, for me to submit a new PR undoing the previous one, to leave it as-is, or something else?
